packages/react-components/src/components/DevicePermissions/DomainPermissionsScaffolding.tsx

```diff
@@ -122,7 +122,10 @@ export const DomainPermissionsContainer = (props: DomainPermissionsContainerProp
   const theme = useTheme();
 
   return (
-    <Stack style={{ padding: '2rem', maxWidth: '25.375rem' }} aria-label={strings?.ariaLabel}>
+    <Stack
+      style={{ padding: '2rem', maxWidth: '25.375rem', maxHeight: '100vh', alignSelf: 'center', overflow: 'hidden' }}
```

Comment:
Tried with % the scroll bar reappears , using vh for max height should be ok?

Reply:
`vh` only works when testing locally because the composite is sized to the full screen (i.e. 100vh), if you were to run the sample with the composite sized to something like 80vh, would `max-height: 100vh` still prevent the scrollbar from showing?

Reply:
100vh was used to work together with overflow hidden to remove the scrollbars, but yeah I didn't notice that a big portion of the modal did got cut off when overflow is set to hidden. Thought only a tiny bit of padding got cut off. 

The reason behind this is we have hard-coded margin /paddings and the landscape size is just too small for them to fit in the viewport. In this case should we keep the scroll bars?